src/util/dsp/window.cpp

```diff
@@ -12,7 +12,7 @@ namespace otto::util::dsp {
   void Window::reset (int size, WindowType type, bool normalize) noexcept
   {
       window_buffer.resize (size);
-      compute (gsl::as_span(window_buffer.begin(), size), type, normalize);
+      compute (gsl::span(window_buffer.begin(), size), type, normalize);
```

Comment:
To be honest, using a span here might not make too much sense, since its a private function anyway, right?

This is great though, im merging it